Core/src/main/java/com/paypal/android/core/graphql/common/Extension.kt

```diff
@@ -0,0 +1,6 @@
+package com.paypal.android.core.graphql.common
+
+data class Extension(
```

Comment:
This class/name is a little unclear to me - is this just for extra error info returned from GraphQL?